ci/ray_ci/automation/BUILD.bazel

```diff
@@ -162,3 +162,13 @@ py_binary(
         ci_require("click"),
     ],
 )
+
+py_binary(
+    name = "list_docker_tags",
```

Comment:
why do you need a binary?

like for check, can it be just a python program?

Reply:
the python file imports modules from `ci.ray_ci` so I turned it into a binary to work with dependency